dashboard/Dockerfile

```diff
@@ -1,8 +1,12 @@
-# Copyright (c) 2012-2017 Red Hat, Inc
-# All rights reserved. This program and the accompanying materials
-# are made available under the terms of the Eclipse Public License v1.0
-# which accompanies this distribution, and is available at
-# http://www.eclipse.org/legal/epl-v10.html
+# Copyright (c) 2015-2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
```

Comment:
FYI I'm not sure we need the Contributors: section in EPL 2

Reply:
I've just updated these header as it has been done in `*.ts` files.
@skabashnyuk what do you think?

Reply:
I'm not sure too